Sources/PayPalDataCollector/PayPalDataCollector.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import PPRiskMagnes
+
+public class PayPalDataCollector {
+
+    private let magnesSDK: MagnesSDKProtocol
+    private let deviceInspector: DeviceInspectorProtocol
+
+    init(magnesSDK: MagnesSDKProtocol = MagnesSDK.shared(), deviceInspector: DeviceInspectorProtocol = DeviceInspector()) {
+        self.magnesSDK = magnesSDK
+        self.deviceInspector = deviceInspector
+    }
+
+    public func collectDeviceData(additionalData: [String: String] = [:]) -> String {
+        let deviceIdentifier = deviceInspector.paypalDeviceIdentifier()
+        let params = MagnesSetupParams(
+            env: .LIVE,
+            appGuid: deviceIdentifier,
+            apnToken: "",
+            isRemoteConfigDisabled: false,
+            isBeaconDisabled: false,
+            source: .BRAINTREE
```

Comment:
Should we use a different source for this SDK?

Reply:
Yeah def went with `.PAYPAL`.